.github/CHANGELOG.md

```diff
@@ -87,6 +87,7 @@
 1. [LIGHTS] Increased max brightness of cockpit displays - @frankkopp (Frank Kopp)
 1. [EFB] Improved Aircraft Presets procedures - @frankkopp (Frank Kopp)
 1. [FMGC] Make DES SPD LIM available prior to descent - @tracernz (Mike)
+1. [ENGINE] Refactor FADEC code for A32NX & A380X - @Taz5150 (TazX [Z+2]#0405)
```

Comment:
0.9 has been and gone. Need to target 0.10. But... does this need a changelog? If it doesn't change anything user-facing then I'd say no.

Reply:
Yes, this will be changed to 0.10 but should be there since it possibilitates the A380

Reply:
If it doesn't make a user visible change there's no point adding noise to the changelog. We have git history for code changes.

Reply:
Understood and changed.

Thank you!